.eslintrc

```diff
@@ -2,8 +2,6 @@
   "extends": "vtex",
   "root": true,
   "env": {
-    "node": true,
-    "es6": true,
-    "jest": true
```

Comment:
I don't think this should be changed.

Reply:
Thanks, I updated the file again. They were removed automatically after I ran vtex setup